app/collections/show-list.cjsx

```diff
@@ -108,6 +108,7 @@ module?.exports = React.createClass
             <div className="collection-subjects-list">{subjects.map(subjectNode)}</div>
 
             <Paginator
+              collection={@props.collection}
```

Comment:
I think this should be added to the `propTypes` and default props in the Paginator component too.